google/ads/google_ads/client.py

```diff
@@ -418,3 +593,42 @@ def _validate_login_customer_id(login_customer_id):
             raise ValueError('The specified login customer ID is invalid. '
                              'It must be a ten digit number represented '
                              'as a string, i.e. "1234567890"')
+
+
+def _parse_metadata_to_json(metadata):
+    """Parses metadata from a gRPC requests and responses to a JSON string.
+    Obscures the value for "developer-token".
+
+    Args:
+        metadata: a list of tuples of metadata information from a
+        gRPC response
```

Comment:
**nitpick**: Indent to indicate continuation of comment.

Reply:
Fixed.